ports/psoc6/modules/machine/machine_pin.c

```diff
@@ -21,13 +21,13 @@ typedef struct _machine_pin_io_obj_t {
     uint8_t pull;
 } machine_pin_io_obj_t;
 
-#define MAX_PIN_IO MP_ARRAY_SIZE(machine_pin_phy_obj)
-machine_pin_io_obj_t *pin_io[MAX_PIN_IO] = {NULL};
+
+machine_pin_io_obj_t *pin_io[MAX_IO_PINS] = {NULL};
```

Comment:
mmm MAX_IO_PINS is gone or=?

Reply:
MAX_IO_PINS is now defined in generated sources for boards. Will be in build.